src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -155,14 +155,6 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
       }
     }
   }
-
-  /** Resets fields when returned to the Tabs component */
-  @Watch('getDisplayedComponent')
-  watchDisplayedComponent (displayedComponent: string) {
-    if (displayedComponent === 'Tabs') {
-      this.onBusiness(null)
-    }
-  }
```

Comment:
This is no longer needed because of the fix in #690

Reply:
Can the import of getDisplayedComponent be removed too?

Reply:
The import is in the SearchMixin, and it's used in request-action.vue